css/pages/home.css

```diff
@@ -266,17 +267,28 @@
   margin-bottom: 20px;
 }
 .footer-links > li,
-.footer-links > li > a {
+.footer-links > li > a,
+.footer-links > li > p > a {
   text-decoration: none;
   list-style-type: none;
-  padding: 5px 0 5px 0;
-  display: block;
+  padding-bottom: 9px;
+  color: var(--bg-color-secondary-font);
 }
-
-.footer-links > li > i,
-.footer-links > li > p {
-  display: inline;
+.footer-links > li {
+  display: flex;
 }
+.footer-links > li > a:hover,
+.footer-links > li > p > a:hover {
+  color: var(--bg-color-green);
+  margin-left: 5px;
+  transition: 0.7s;
+}
+.footer-links > li > a:not(:hover),
+.footer-links > li > p > a:not(:hover) {
```

Comment:
Kuo skiriasi `.footer-links > li > a:not(:hover)` rašymas, nuo tiesiog jo nerašymo ir turėjimo kaip default stiliaus, t.y. tiesiog kaip `.footer-links > li > a`?🤔

Reply:
nuemus pele, transition i pradinia spalva

Reply:
pataisiau be not(:hover)